src/frontend/screens/WineManager/index.tsx

```diff
@@ -49,7 +49,7 @@ export default function WineManager(): JSX.Element | null {
       }
     }
 
-    return refreshWineVersionInfo(true)
+    return () => refreshWineVersionInfo(true)
```

Comment:
The only issue I found after the upgrade was this one. React 18 complains that `useEffect `can only return a clean-up function so it was failing. Needed to change this approach to fix the issue.

Reply:
I doubt that's what was intended here, seems more like
```suggestion
    refreshWineVersionInfo(true)
```
would be correct

CC @Nocccer

Reply:
Yes, in practice wont make a difference. The thing is that returning a cleanup function like I did makes the method being called at the end of the `useEffect `always.